CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ Looking for changes that affect our C API? See the [C API Changelog](lib/c-api/C
 ## **Unreleased**
 
 ### Changed
+- [#3004](https://github.com/wasmerio/wasmer/pull/3004) Remove cranelift from wasi/sys-default feature, making it possible to select the compiler backend when depending on wasmer-wasi
```

Comment:
Put this change in the same commit as the other, no need for extra "Add CHANGELOG entry" commits